test/aulength.c

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file src/aulength.c audio file duration test
+ *
+ * Copyright (C) 2023 Lars Immisch
+ */
+
+#include <re.h>
+#include <rem.h>
+#include "test.h"
+
+
+#define DEBUG_MODULE "aulength"
+#define DEBUG_LEVEL 5
+#include <re_dbg.h>
+
+
+int test_aulength(void)
+{
+	struct aufile *af;
```

Comment:
initialize with NULL here. So mem_deref is fine if aufile_open fails.

Reply:
Good point